### ISSUE TITLE
Offensive server name 'MiNET'

### ISSUE THREAD

**Issue by nabokov-ef** [NiclasOlofsson/MiNET#446]
Hello!
I understand you did not know when you chose the name for the server, but the Russian word 'minet' literally means a BJ (https://www.urbandictionary.com/define.php?term=minet). For 150m Russian people this word is obscene. When we are speaking about a computer game without age restriction, this situation is getting worse.
To illustrate the problem I corrected some issues titles:
-- No logging when BJ is started from a remote location
-- BJ won't start or compile under linux
-- BJ service is now running, press Control+C to exit

Would you mind renaming the server and the repository?

**Comment by dktapps** [comment]
This project has existed for something like 4 years and you're hardly the first person to notice this. I'm sure you can find a language where the words I'm writing are obscene to someone, somewhere.

This issue reminds me of this one: https://jira.atlassian.com/browse/JRASERVER-65811

**Comment by dktapps** [comment]
Please also ask these guys to change their name: https://reports.ofsted.gov.uk/provider/21/102392
It's in Middle**sex** so this is a really terrible issue for the little kids.

**Comment by kennyvv** [comment]
Also, what kid (that actually laughs about stuff like that) is gonna need to use MiNET personally. I'd say its usually older people actually setting everything up. Especially with software like MiNET... Sure there is gonna be a few exceptions, but i wouldn't say a few immature laughs are enough to change the name of a whole project...

**Comment by HypertextPP** [comment]
> Would you mind renaming the server and the repository?

This is just as ludicrous as asking [Siemens](https;//twitter.com/Siemens) (the largest manufacturing company in Europe) to change their name purely because its English pronunciation is the same as that of the male reproductive fluid – approximately 1.5 billion people speak English.

**Comment by nabokov-ef** [comment]
Ok, at least I told about that.
Just another example of an ambivalent name: https://wiki.cdot.senecacollege.ca/wiki/Pidora_Russian

**Comment by NiclasOlofsson** [comment]
Yeah, thanks for the feedback @nabokov-ef. At the time of naming it wasn't known (since I don't speak French or Russian). I did notice it when Azure refused to let me name a virtual server to MiNET (who on Microsoft decided to put profanity filters on server names?!). It's not good, I admit it is bad, but it's been a long standing joke in the community for a long time, and changing the name would create so much problems it simply wouldn't work.

**Comment by Tarik02** [comment]
Duplicate https://github.com/NiclasOlofsson/MiNET/issues/64

**Comment by Tarik02** [comment]
Just pronounce this as "майнет" (mainet) and everything will be ok.

**Comment by gimdh** [comment]
I do understand changing project name is difficult. However, whenever I search for MiNet in basically any search engine, it freaks out and safe search filter kicks in.
I suggest using some other "Community name" for searching and documentation like Go did - GoLang. Something like MineNet or MiDotNet or so would be helpful.

**Comment by nnsense** [comment]
Hi :) Interesting ticket :D I'm Italian. Remember about SEGA, one of the most famous game consolle during 80/90s? Well that LITERALLY, without any translitteration, means.. well..  _enjoying youself__.. it's a very vulgar word. A website in US about christianity and family was called figa, which in italian is.. errrr... female.. well, there are plenty of this examples, I don't think is a big issue after all :)

**Comment by JonMagon** [comment]
As an Russian, I don't agree with you. Try to pronounce /mai.net/. There's nothing wrong with knowing what blowjob is. The sexual education in our education system is lacking, and that's really bad. Let this repo give kids a chance when they go on the Internet, before they will do it behind the garage.

**Comment by IrregularShed** [comment]
I'm reminded of the program that the Territorial Army (basically British army reserves) started in about 2001, "Time to Get FitTA" which was accompanied by a website, `getfitta.co.uk`, which I suspect @NiclasOlofsson will understand led to more visits coming from giggling Swedish teenagers than anyone from the UK 😀

**Comment by SPAWNRYS-ban** [comment]
https://github.com/NiclasOlofsson/MiNET/issues/635
